Update README.md (#5)
skills/github-pages: README.md

1. Under your repository name, click **Settings**.
1. In the "GitHub Pages" section, use the Source drop-down, then select **main branch**.
1. Wait about _one minute_, then refresh this page for the next step.

>    > Turning on GitHub Pages creates a deployment of your repository. It may take up to a minute to respond as it awaits the deployment. Future steps will be about 20 seconds; this step is slower.

Reviewer: ```suggestion
   > Turning on GitHub Pages creates a deployment of your repository. GitHub Actions may take up to a minute to respond while waiting for the deployment. Future steps will be about 20 seconds; this step is slower.
```
